src/elements/container/sticky-bar/sticky-bar.visual.spec.ts

```diff
@@ -21,7 +21,7 @@ import '../../title.js';
 
 describe(`sbb-sticky-bar`, () => {
   const cases = {
-    color: [undefined, 'white', 'milk'],
+    color: [undefined, 'white', 'milk', 'midnight', 'charcoal'],
```

Comment:
I suggest to adapt link color in tests, and maybe a test combining a dark container with a dark sticky bar (probably the default case).

Reply:
I added cases with the two components in the same color